watcher/lib/influxdb/rollups.go

```diff
@@ -119,38 +119,68 @@ func buildQuery(r Rollup) (string, error) {
 	return b.String(), nil
 }
 
+// funcsWithParams define which functions need an additional parameter
```

Comment:
usually you don't differentiate singular/plural for variable/function names and always use 3rd person in comments:
```go
// funcsWithParams defines which functions ...
```